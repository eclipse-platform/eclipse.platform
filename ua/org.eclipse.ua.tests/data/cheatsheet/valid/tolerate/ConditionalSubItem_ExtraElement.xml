<?xml version="1.0" encoding="UTF-8" ?> 
<!--
     Copyright (c) 2005, 2006 IBM Corporation and others.

     This program and the accompanying materials
     are made available under the terms of the Eclipse Public License 2.0
     which accompanies this distribution, and is available at
     https://www.eclipse.org/legal/epl-2.0/

     SPDX-License-Identifier: EPL-2.0
    
     Contributors:
         IBM Corporation - initial API and implementation
 -->

<cheatsheet title="Test cheatsheet without cheatsheet element">

	<intro>
		<description>
Simple test
		</description>
</intro>

	<item
		title="Just a simple test">
		<description>
Did it work? Of course not, it is suppose to cause a parsing error.
		</description>
	<conditional-subitem condition="${pattern}">
		<extra>testing</extra>
		<subitem
			when="Factory"
			label="Customize the Factory Pattern"
			skip="true">
			<action
				confirm="true"
				pluginId="org.eclipse.ua.tests"
				class="org.eclipse.ui.examples.cheatsheets.pattern.actions.FactoryCustomizeAction"/>
		</subitem>
		<subitem
			when="Singleton"
			label="Customize the Singleton Pattern"
			skip="false">
			<action
				confirm="true"
				pluginId="org.eclipse.ua.tests"
				class="org.eclipse.ui.examples.cheatsheets.pattern.actions.SingletonCustomizeAction"/>
		</subitem>
		<subitem
			when="Visitor"
			label="Customize the Visitor Pattern"
			skip="true">
			<action
				confirm="false"
				pluginId="org.eclipse.ua.tests"
				class="org.eclipse.ui.examples.cheatsheets.pattern.actions.VisitorCustomizeAction"/>
		</subitem>
	</conditional-subitem>
</item>

</cheatsheet>
