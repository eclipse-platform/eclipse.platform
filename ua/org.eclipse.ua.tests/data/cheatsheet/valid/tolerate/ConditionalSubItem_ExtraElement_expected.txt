<?xml version="1.0" encoding="UTF-8" ?>
<cheatsheet
      title="Test cheatsheet without cheatsheet element">
   <item
         title="Introduction"
         description="Simple test"
         Href="null"
         contextId="null">
      <nullAction/>
      <nullList/>
      <nullPerformWhen/>
      <nullList/>
   </item>
   <list>
      <item
            title="Just a simple test"
            description="Did it work? Of course not, it is suppose to cause a parsing error."
            Href="null"
            contextId="null">
         <nullAction/>
         <list/>
         <nullPerformWhen/>
         <list>
            <conditionalSubItem
                  condition="${pattern}">
               <list>
                  <subItem
                        label="Customize the Factory Pattern"
                        when="Factory"
                        isSkip="true">
                     <action
                           class="org.eclipse.ui.examples.cheatsheets.pattern.actions.FactoryCustomizeAction"
                           pluginId="org.eclipse.ua.tests"
                           when="null"
                           isConfirm="true">
                        <nullArray/>
                     </action>
                     <nullPerformWhen/>
                  </subItem>
                  <subItem
                        label="Customize the Singleton Pattern"
                        when="Singleton"
                        isSkip="false">
                     <action
                           class="org.eclipse.ui.examples.cheatsheets.pattern.actions.SingletonCustomizeAction"
                           pluginId="org.eclipse.ua.tests"
                           when="null"
                           isConfirm="true">
                        <nullArray/>
                     </action>
                     <nullPerformWhen/>
                  </subItem>
                  <subItem
                        label="Customize the Visitor Pattern"
                        when="Visitor"
                        isSkip="true">
                     <action
                           class="org.eclipse.ui.examples.cheatsheets.pattern.actions.VisitorCustomizeAction"
                           pluginId="org.eclipse.ua.tests"
                           when="null"
                           isConfirm="false">
                        <nullArray/>
                     </action>
                     <nullPerformWhen/>
                  </subItem>
               </list>
            </conditionalSubItem>
         </list>
      </item>
   </list>
</cheatsheet>
